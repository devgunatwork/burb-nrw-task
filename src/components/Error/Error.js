import React from 'react';
import PropTypes from 'prop-types';
import Typography from '@material-ui/core/Typography';

import styles from './Error.module.scss';

const Error = ({ message }) => (
  <div className={styles.root}>
    <Typography variant="h4">{message}</Typography>
  </div>
);

Error.propTypes = {
  message: PropTypes.string.isRequired,
};

export default Error;
