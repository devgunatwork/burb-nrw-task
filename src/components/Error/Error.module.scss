@import "src/style/variables/variables";
@import "src/style/mixins/mixins";

.root {
  @include pageCenter;
  color: $error-severe-color;
}
