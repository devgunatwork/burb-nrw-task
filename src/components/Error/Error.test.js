import React from 'react';
import Error from './Error';

describe('components', () => {
  describe('Error', () => {
    describe('UI', () => {
      it('should match the snapshot', () => {
        const wrapper = shallow(<Error message="Failed to fetch" />);

        expect(wrapper).toMatchSnapshot();
      });
    });
  });
});
