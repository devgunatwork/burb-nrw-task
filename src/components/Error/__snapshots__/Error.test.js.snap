// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components Error UI should match the snapshot 1`] = `
<div
  className="root"
>
  <WithStyles(ForwardRef(Typography))
    variant="h4"
  >
    Failed to fetch
  </WithStyles(ForwardRef(Typography))>
</div>
`;
